cea/demand/calibration/bayesian_calibrator/calibration_gaussian_emulator.py

```diff
@@ -33,6 +32,18 @@ def gaussian_emulator(locator, samples, cv_rmse, building_name):
            file with database of emulator stored in locator.get_calibration_cvrmse_file(building_name)
 
     """
+    # INITIALIZE TIMER
+    t0 = time.clock()
+
+    # Local variables
+    building_name = config.single_calibration.building
+    building_load = config.single_calibration.load
+    with open(locator.get_calibration_problem(building_name, building_load),'r') as input_file:
+        problem = pickle.load(input_file)
+    samples_norm = problem["samples_norm"]
+    rmse = problem["cv_rmse"]
+    import pandas
+    pandas.DataFrame({'cvrmse':problem["cv_rmse"], 'rmse':problem["rmse"]}).to_csv(r'C:\Users\JimenoF\Desktop/test3.csv')
 
```

Comment:
Prevent hard coded csv file names. Link this to the inputlocator

Reply:
fixed